tests/fixtures/ci/docker/alpine.Dockerfile

```
@@ -8,34 +8,38 @@
 FROM alpine:3.21
 
 # === SECTION: base packages (managed) ===
-# Source: template (distro-specific package list)
+# Source: template (distro-specific seed contract)
+# The seed is the whole host contract: curl fetches mise, git is what uv shells
+# out to for the flext-infra git+https requirement (uv embeds no git of its
+# own), and make invokes the verbs. bash is for the project scripts.
+# libstdc++ is musl-specific: node and bun link against libstdc++.so.6 at
+# runtime, so on Alpine they install cleanly and then fail with "Error loading
+# shared library". Glibc images ship it in the base system.
 RUN apk add --no-cache \
```

Hadolint (2.15.1)

[warning] Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`

(DL3018)